02 Writing Algorithms/16 Importing Data/02 Streaming Data/02 Custom Securities/02 CSV Format Example/04 Define Custom Types.html

```diff
@@ -1,4 +1,4 @@
-<p>To define a custom data type, inherit the <code class="csharp">BaseData</code><code class="python">PythonData</code> class and override the <a href="/docs/v2/writing-algorithms/importing-data/streaming-data/key-concepts#04-Set-Data-Sources">GetSource</a> and <a href="/docs/v2/writing-algorithms/importing-datacustom-securities/key-concepts#05-Parse-Custom-Data">Reader</a> methods.</p>
+<p>To define a custom data type, inherit the <code class="csharp">BaseData</code><code class="python">PythonData</code> class and override the <a href="/docs/v2/writing-algorithms/importing-data/streaming-data/key-concepts#04-Set-Data-Sources">GetSource</a> and <a href="/docs/v2/writing-algorithms/importing-data/custom-securities/key-concepts#06-Receive-Custom-Data">Reader</a> methods.</p>
```

Comment:
This should be /docs/v2/writing-algorithms/importing-data/streaming-data/key-concepts#05-Parse-Custom-Data

Reply:
Same with the 3 other links in this PR